src/components/ReviewCard/ReviewCard.jsx

```diff
@@ -88,7 +88,7 @@ HeaderReview.propTypes = {
 SubheaderReview.propTypes = {
     createdAt: PropTypes.string.isRequired,
     jobTitle: PropTypes.string.isRequired,
-    isStillWorkingHere: PropTypes.string.isRequired,
+    isStillWorkingHere: PropTypes.number.isRequired,

```

Comment:
You should use:
`
isStillWorkingHere: PropTypes.oneOf([0, 1]).isRequired,
`
We only expect two numbers, 0 or 1, it would be better to specify which numbers we are going to be getting